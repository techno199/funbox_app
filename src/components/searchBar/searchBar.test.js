import React from 'react'
import { SearchBar } from '.'
import renderer from 'react-test-renderer'

test('should render search box', () => {
  const component = renderer.create(
    <SearchBar value={'Some value'} />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
