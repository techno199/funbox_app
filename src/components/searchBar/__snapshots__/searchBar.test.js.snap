// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render search box 1`] = `
<div
  className="SearchBarUnstyled-root-1"
>
  <div
    className="MuiPaper-root-4 MuiPaper-elevation1-7 MuiPaper-rounded-5 SearchBarUnstyled-paper-2"
  >
    <div
      className="MuiInputBase-root-31 SearchBarUnstyled-input-3"
      onClick={[Function]}
    >
      <input
        className="MuiInputBase-input-41"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        placeholder="Search Google Maps"
        type="text"
        value="Some value"
      />
    </div>
  </div>
  <div
    className="MuiPaper-root-4 MuiPaper-elevation1-7 MuiPaper-rounded-5 SuggestionsUnstyled-root-48"
    hidden={true}
  >
    <ul
      className="MuiList-root-49 MuiList-padding-50"
    />
  </div>
</div>
`;
